tests/inference/test_utils.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import torch
+
+from utils import get_deployment_actor_options, StoppingCriteriaSub, max_input_len, get_torch_dtype, is_cpu_without_ipex
+from inference_config import InferenceConfig, DEVICE_CPU
+
+# Mock the InferenceConfig for testing
+@pytest.fixture
+def mock_infer_conf():
```

Comment:
Can we make it configurable? so that ipex and deepspeed can sometimes be enabled.

Reply:
i'm writting more test case with utils.py， need i add everything before PR or complete the simple framework merge first

Reply:
ok,i will make it configurable